src/transform.ts

```diff
@@ -96,29 +87,19 @@ export function opacity(o: number): Transform {
 export function compose(t1: Transform, t2: Transform): Transform {
   return transform(
     point.addPoints(t1.translate, t2.translate),
-    anchor.lift2((a, b) => a + b)(t1.rotate, t2.rotate),
-    anchor.lift2(point.mulPoints)(t1.scale, t2.scale),
-    t1.opacity * t2.opacity
+    anchor.lift2((a: Deg, b: Deg) => a + b)(t1.rotate, t2.rotate),
+    anchor.lift2((a: Point, b: Point) => point.mulPoints(a, b))(t1.scale, t2.scale),
+    t1.opacity * t2.opacity,
   );
 }
 
 export function composeAll(ts: Array<Transform>): Transform {
   return ts.reduce(compose, reset);
 }
 
-function eqNum(a, b) {
-  return Math.round(a) === Math.round(b);
-}
-
 export function equals(a: Transform, b: Transform): boolean {
-  return eqNum(a.translate.x, b.translate.x) &&
-      eqNum(a.translate.y, b.translate.y) &&
-      eqNum(a.rotate.anchor.x, b.rotate.anchor.x) &&
-      eqNum(a.rotate.anchor.y, b.rotate.anchor.y) &&
-      eqNum(a.rotate.value, b.rotate.value) &&
-      eqNum(a.scale.anchor.x, b.scale.anchor.x) &&
-      eqNum(a.scale.anchor.y, b.scale.anchor.y) &&
-      eqNum(a.scale.value.x, b.scale.value.x) &&
-      eqNum(a.scale.value.y, b.scale.value.y) &&
-      eqNum(a.opacity, b.opacity);
+  return eqPoint(a.translate, b.translate) && 
+    eqAnchor(a.rotate, b.rotate) &&
+    eqAnchor(a.scale, b.scale) &&
```

Comment:
I dont wanna make more work for the sake of it, but maybe it would be more readable if its like
function eqTransform contains functions eqAchor and eqValue
caus eqAnchor currently also checks eqValue as well?

Reply:
I genuinely can't parse this comment, what do you mean?

Reply:
Ah nm i think i was just confused about what that func did